src/load_fund_data.py

```diff
@@ -0,0 +1,233 @@
+from configs.constants import TableNames
+from datetime import datetime
+from models.external_fund import ExternalFund
+import logging
+import os
+import pandera as pa
+from pandera.dtypes import (
+    String,
+    Float,
+    Int,
+    Bool,
+    DateTime,
+)
+import polars as pl
+import re
+from utils.utils import Utils
+
+
+def get_config_n_constant() -> tuple[str, str, str]:
+    """
+    Retrieves:
+        - Database and input data paths from config
+        - Fund holdings table name from constants.
+
+    Returns:
+        tuple[str, str, str]: A tuple containing the database path, input data path, and the table name for fund holdings data.
+    """
+    config = Utils.load_config()
+    database_path = config.database_path
+    input_data_path = config.input_data_path
+    fund_holdings_table_name = TableNames.FUND_HOLDINGS
+    return (database_path, input_data_path, fund_holdings_table_name)
+
+
+def clean_fund_name(fund_name_raw: str) -> str:
+    """
+    Cleans the fund name by removing unwanted characters and converting it to Title case.
+    Assumes that the fund name is the last part of the string after splitting by spaces or special characters.
+    e.g. "TT_monthly_Trustmind", "rpt-Catalysm", "Fund Whitestone", "Applebead"
+
+    Args:
+        fund_name_raw (str): The raw fund name.
+
+    Returns:
+        str: The cleaned fund name.
+    """
+    split_name = re.split(r"[\s\-_.]+", fund_name_raw)
+    end_part = split_name[-1].strip() if split_name else None
+    return end_part.title() if end_part.isalnum() else None
```

Comment:
You'll get an exception here if end_part is None from the previous line.
Also, returning None doesn't fit the typing - would you get a TypeError elsewhere?